packages/theme/webpack.config.js

```diff
@@ -23,16 +23,10 @@ module.exports = (env, argv) => {
 			rules: [
 				{
 					test: /\.woff(2)?(\?[a-z0-9=&.]+)?$/,
-					use: [
-						{
-							loader: require.resolve('file-loader'),
-							options: {
-								outputPath: 'fonts',
-								name: '[name].[ext]',
-								esModule: false,
-							},
-						},
-					],
+					type: 'asset/resource',
```

Comment:
adapt to breaking changes of css-loader 6:
https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#notes